### PR TITLE
feat(Mutate file only): Specify what files to mutate

### PR DESCRIPTION
**Pull request by richardwerkman**
This PR aims to improve file exclusion. It works as described in #322.

### CODE REVIEW

File: src/Stryker.CLI/Stryker.CLI/CLIOptions.cs [hunk 1]
```diff
@@ -90,17 +90,16 @@ public static class CLIOptions
         public static readonly CLIOption<int> MaxConcurrentTestRunners = new CLIOption<int>
         {
             ArgumentName = "--max-concurrent-test-runners",
-            ArgumentShortName = "-m <integer>",
+            ArgumentShortName = "-c <integer>",
```
Comment (Mobrockers) [inline src/Stryker.CLI/Stryker.CLI/CLIOptions.cs]: This is super breaking, we should consider this carefully and I would prefer to do this in a separate pr so we can go over and fix all flags we deem wrong at once.

**Comment by richardwerkman** [comment]
This should change to getting the config from the project under test. This has to do with the future plans described in #459
